source/state_representation/include/state_representation/space/cartesian/CartesianTwist.hpp

```diff
@@ -47,6 +47,9 @@ class CartesianTwist : public CartesianState {
   void set_force(const Eigen::Vector3d& force) = delete;
   void set_torque(const Eigen::Vector3d& torque) = delete;
   void set_wrench(const Eigen::Matrix<double, 6, 1>& wrench) = delete;
+  CartesianState inverse() const = delete;
```

Comment:
Contrary to what @buschbapti mentioned on slack, I believe the inverse function still makes sense for the twist and the wrench. In the absence of any pose orientation, a pure twist inverse would simply be the negation of the linear and angular vectors (plus swapping the reference frame). But, as with the pose, you can still call the base CartesianState::inverse and downcast the return.